build/Settings.cs

```diff
@@ -59,9 +59,9 @@ public static class Settings
 
         public static string ExtensionBundleVersionRange = "[3.*, 4.0.0)";
 
-        public static string ExtensionBundleBuildVersion = "3.0.0";
+        public static string ExtensionBundleBuildVersion = "3.1.0";
 
-        public static string TemplatesVersion = "2.0.1619";
+        public static string TemplatesVersion = "2.0.1635";

```

Comment:
@soninaren Should I revert both of these changes for bundles 3.x if this change is expected to go as part of 3.0.0 (assuming a new template release will be cut) ? Or should I keep just the `TemplatesVersion` change alone and keep ExtensionBundleVersion at 3.0.0 ?